README.md

```diff
@@ -214,7 +214,7 @@ Happy Hacking !
 Simply copy and paste a URL into the provided area. Then click shorten URL! Your URL has now been shortened!
 
 <div align="center">
-<img src="docs/reduced-to.gif" width="600" height="254">
+<img src="assets/images/usage.gif" width="600" height="254">
```

Comment:
Why did you changed it?
Make sure your branch is aligned with the master.

Reply:
It should align with the master branch now.

Reply:
It's still not aligned with the master branch.